src/Helpers/Group/GroupHelper.js

```diff
@@ -1,26 +1,42 @@
 
-import { getApprovalApi } from '../Shared/userLogin';
+import { getGroupApi } from '../Shared/userLogin';
 
-const userApi = getApprovalApi();
+const groupApi = getGroupApi();
 
 export async function fetchGroups() {
-  let groups = await userApi.fetchGroups();
+  let groupsData = await groupApi.listGroups();
+  let groups = groupsData.data;
+  console.log('Groups: ', groups);
   let len = groups.length;
   for (let idx = 0; idx < len; idx++) {
-    let users = await userApi.fetchUsersByGroupId(groups[idx].id);
-    groups[idx].members = users;
+    let groupWithUsers = await groupApi.getGroup(groups[idx].uuid);
+    groups[idx].members = groupWithUsers.principals;
   }
+
   return groups;
 }
 
 export async function updateGroup(data) {
-  await userApi.updateGroup(data.id, data);
+  await groupApi.updateGroup(data.uuid, data);
+  let members_list = data.members.map(user => { return user.username; });
```

Comment:
You can return the value without the block. Also i don't see the `members_list` variable ever change so you should use `const` instead of `let`. And linter will probably complain about not using camel calse
```javascript
const membersList = data.members.map(user => user.username);
```